### PR TITLE
19 create a get route to get our user from db

### LINKED ISSUE #19: Create a GET Route to get our user from DB
We need to have a route to match the name up to a user in the DB

### PR DESCRIPTION
**Pull request by micah-peautagi**
Created a route to get users from database.

closes #19

### CODE REVIEW

**Review by emilyparkes — APPROVED** [review]

File: server/server.js [hunk 1]
```diff
@@ -4,12 +4,14 @@ const express = require('express')
 const server = express()
 
 const starterRoute = require('./routes/starter')
+const users = require('./routes/usersRoutes')
```
Comment (emilyparkes) [inline server/server.js]: Could you please change this line to `const usersRoutes = require('./routes/users')`
Thanks!

File: server/server.js [hunk 2]
```diff
@@ -4,12 +4,14 @@ const express = require('express')
...
 const cohortRoutes = require('./routes/cohortRoutes')
 
 server.use(express.json())
 server.use(express.static(path.join(__dirname, './public')))
 
 server.use('/', starterRoute)
+server.use('/api/v1/users', users)
```
Comment (emilyparkes) [inline server/server.js]: Could you please change this line to `server.use('/api/v1/users', usersRoutes)`
Thanks!

**Review by emilyparkes — CHANGES_REQUESTED** [review]

File: server/server.js (hunk 1 above)
Comment (craftedcraft) [reply server/server.js]: Thanks @emilyparkes , this is now complete

File: server/server.js (hunk 2 above)
Comment (craftedcraft) [reply server/server.js]: Thanks @emilyparkes this is now compete

**Review by emilyparkes — APPROVED** [review]

**Pull request merged**